torch/_dynamo/allowed_functions.py

```diff
@@ -276,7 +276,29 @@ def _builtin_constant_ids():
     return rv
 
 
+# A subcheck of is_allowed, we utilize this for patching is_allowed around distributed.
+# We do this because we want to allow these to be traced, and hence covered in skipfiles, but we do not want them to
+# become TorchVariable
+def _is_allowed_distributed(obj):
+    if not torch.distributed.is_available():
+        return True
+    if obj in [torch.distributed._functional_collectives.all_gather_tensor]:
```

Comment:
if the point of this is to make us inline instead of ingraph-ing, shouldn't this include all the functional collectives?  if so i will look for an existing list or we should make a list that keeps track of them all in functional_collectives.py so dynamo can just reference that list and pt-d can keep it updated

Reply:
We don't need this check, it should be subsumed by the below. I meant to remove it.